### PR TITLE
Format Python Solution for Shell Game (Bronze)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
all test cases pass for this solution :)

### CODE REVIEW

File: solutions/usaco-891.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		points2 += 1
+
+	# if pebble is under third shell
+	lay3[swaps[i][0]], lay3[swaps[i][1]] = lay3[swaps[i][1]], lay3[swaps[i][0]]
+
+	if lay3[swaps[i][2]]:
+		points3 += 1
+
+with open('shell.out', 'w') as fout:
+	fout.write(str(max(points1, points2, points3)))
```
Comment (SansPapyrus683) [inline solutions/usaco-891.mdx:58]:
```suggestion
print(max(points1, points2, points3), file=open('shell.out', 'w'))
```

File: solutions/usaco-891.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	
+	if lay2[swaps[i][2]]:
+		points2 += 1
+
+	# if pebble is under third shell
+	lay3[swaps[i][0]], lay3[swaps[i][1]] = lay3[swaps[i][1]], lay3[swaps[i][0]]
+
+	if lay3[swaps[i][2]]:
+		points3 += 1
+
```
Comment (SansPapyrus683) [inline solutions/usaco-891.mdx:56]: 25, 32, 33, 35, 37, 44, 47, 50, and 53 are unnecessary

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by dongliu0426** [comment]
@ryanchou-dev isn't this the focus problem of the simulation module? https://usaco.guide/bronze/simulation?lang=cpp#example-1

**Comment by ryanchou-dev** [comment]
> @ryanchou-dev isn't this the focus problem of the simulation module? https://usaco.guide/bronze/simulation?lang=cpp#example-1

oh

**Comment by ryanchou-dev** [comment]
@dongliu0426 should i just add my code to the module?

**Review by dongliu0426 — APPROVED** [review]

**Review by SansPapyrus683 — APPROVED** [review]
this looks codey enough to be valid

**Pull request merged**